google/cloud/sql/connector/connector.py

```diff
@@ -36,12 +37,12 @@ def _get_loop() -> asyncio.AbstractEventLoop:
     return _loop
 
 
-def connect(instance_connection_string, driver: str, **kwargs):
+def connect(instance_conn_str, driver: str, **kwargs):
```

Comment:
nit: if you're gonna shorten, shorten consistently 
```suggestion
def connect(inst_conn_str, driver: str, **kwargs):
```